torchvision/prototype/transforms/_meta.py

```diff
@@ -30,7 +30,9 @@ def __init__(self, dtype: torch.dtype = torch.float32) -> None:
 
     def _transform(self, inpt: features.TensorImageType, params: Dict[str, Any]) -> features.TensorImageType:
         output = F.convert_image_dtype(inpt, dtype=self.dtype)
-        return output if features.is_simple_tensor(inpt) else features.Image.new_like(inpt, output, dtype=self.dtype)  # type: ignore[arg-type]
+        return (
+            output if features.is_simple_tensor(inpt) else features.Image.wrap_like(cast(features.Image, inpt), output)
```

Comment:
Nit: Why did we switched from ignore to cast here?

Reply:
There were a bunch of `mypy` errors and it seems I was overly ambitious. Will revert.